src/connectors/resonate.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+if (location.host.startsWith('beta3')) {
+	Connector.playerSelector = '#player';
+
+	Connector.artistSelector = '#player .metas a';
+
+	Connector.trackSelector = '#player .name';
+
+	Connector.durationSelector = '.playlist-item--playing .duration';
+
+	Connector.getTrackArt = () => {
+		return $('.player-artwork > div').css('background-image').replace('url(', '').replace(')', '').replace(/"/gi, '');
+	};
+
+	Connector.trackArtSelector = '.current-cover';
```

Comment:
This property and another one below won't be used because of overriden `Connector.getTrackArt` function.

Reply:
good catch!